etc/gwenview.profile

```diff
@@ -27,7 +28,7 @@ seccomp
 shell none
 tracelog
 
-private-bin gwenview,kbuildsycoca4,gimp,gimp-2.8
+private-bin gwenview,kbuildsycoca4,kbuildsycoca5,gimp,gimp-2.8
```

Comment:
Thanks for the review @smitsohu . The pull request from @VladimirSchowalter20 dropping kbuildsyscoca5 has just been merged.